### ISSUE TITLE
Error 30, read-only filesystem

### ISSUE THREAD

**Issue by IvanGaravito** [mbusb/multibootusb#56]
Cannot install distro, mbusb outputs the following:

```
Boot flag set to bootable on /dev/sdb                                     
RRIP: rrip_offset 0
Applying Ubuntu patch...
Copying chain.c32
Copying gfxboot.c32
Copying ldlinux.c32
Could not copy ldlinux.c32
Could not copy libcom32.c32
Could not copy libutil.c32
Could not copy vesamenu.c32
RRIP: rrip_offset 0
Updating main syslinux config file...
Traceback (most recent call last):
  File "/usr/bin/multibootusb", line 298, in install_syslinux
    self.update_cfg_files.distro_cfg_files()
  File "/usr/lib/python2.7/dist-packages/scripts/update_cfg.py", line 171, in distro_cfg_files
    self.main_cfg_file()
  File "/usr/lib/python2.7/dist-packages/scripts/update_cfg.py", line 203, in main_cfg_file
    config_file = open(sys_cfg_file, "a")
IOError: [Errno 30] Sistema de archivos de solo lectura: '/media/ivangaravito/MBUSB/multibootusb/syslinux.cfg'
Closing multibootusb...
```
## My system
- OS => Ubuntu Mate 16.04.1
- USB size => 16 GB
- USB FS => FAT32
- MBUSB version => 7.5.0-1
- ISO => Ubuntu 16.04.1 and Ubuntu Gnome 16.04.1
## Details

Before running `multibootusb`, I look at mounted devices:

``` sh
ૐ  » ~ λ mount | grep sdb
/dev/sdb1 on /media/ivangaravito/MBUSB type vfat (rw,nosuid,nodev,relatime,uid=1000,gid=1000,fmask=0022,dmask=0022,codepage=437,iocharset=iso8859-1,shortname=mixed,showexec,utf8,flush,errors=remount-ro,uhelper=udisks2)
```

Next, running multibootusb and trying to install a distro, gives me the error **IOError: [Errno 30] Sistema de archivos de solo lectura: '/media/ivangaravito/MBUSB/multibootusb/syslinux.cfg'** or **IOError: [Errno 30] Read-only filesystem: '/media/ivangaravito/MBUSB/multibootusb/syslinux.cfg'**.

![MultiBootUSB after error](https://cloud.githubusercontent.com/assets/1981344/18759956/93f7b46a-80c4-11e6-8166-8b207355660c.png)

Checking again the mounted devices:

``` sh
ૐ  » ~ λ mount | grep sdb
/dev/sdb1 on /media/ivangaravito/MBUSB type vfat (ro,nosuid,nodev,relatime,uid=1000,gid=1000,fmask=0022,dmask=0022,codepage=437,iocharset=iso8859-1,shortname=mixed,showexec,utf8,flush,errors=remount-ro,uhelper=udisks2)
```

Checking kernel messages:

``` sh
ૐ  » ~ λ dmesg | tail -n 2
[13739.878896] FAT-fs (sdb1): error, fat_free_clusters: deleting FAT entry beyond EOF
[13739.878902] FAT-fs (sdb1): Filesystem has been set read-only
```

**Comment by mbusb** [comment]
Can you try the development version and report back...

**Comment by IvanGaravito** [comment]
Hey guys, I've tried the development version and it creates new distros successfully, but I can get booted from the USB  =/ , it only boots using QEmu

**Comment by mbusb** [comment]
Fixed in the latest release version 8.0.0.

**Comment by khalil2535** [comment]
Got this today :disappointed: 
<code>
Checking boot flag on /dev/sdb
</code><br><code>
Disk /dev/sdb already has boot flag.
</code><br><code>
Updating distro specific config files...
</code><br><code>
Updating multibootusb config file...
</code><br><code>
Traceback (most recent call last):
</code><br><code>
  File "/usr/local/lib/python3.5/dist-packages/scripts/mbusb_gui.py", line 352, in install_syslinux
    config.distro, config.persistence)
</code><br><code>
  File "/usr/local/lib/python3.5/dist-packages/scripts/update_cfg_file.py", line 345, in update_distro_cfg_files
</code><br><code>
    update_mbusb_cfg_file(iso_link, usb_uuid, usb_mount, distro)
</code><br><code>
  File "/usr/local/lib/python3.5/dist-packages/scripts/update_cfg_file.py", line 484, in update_mbusb_cfg_file
</code><br><code>
    config_file = open(sys_cfg_file, "a")
</code><br><code>
OSError: [Errno 30] Read-only file system: '/run/media/root/MJRO1802/multibootusb/syslinux.cfg'
Aborted (core dumped)</code>
